cmd/clusterctl/cmd/rollout.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/lithammer/dedent"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/cluster-api/cmd/clusterctl/cmd/rollout"
+)
+
+var (
+	rolloutLong = LongDesc(`
+		Manage the rollout of a cluster-api resource.` + rolloutValidResources)
+
+	rolloutExample = Examples(`
+		# Force an immediate rollout of machinedeployment
+		clusterctl alpha rollout restart machinedeployment/my-md-0
+
+		# Rollback to the previous revision of a machinedeployment
+		clusterctl alpha rollout undo machinedeployment/my-md-0
+
+		# Check the rollout status of a machinedeployment
+		clusterctl alpha rollout status machinedeployment/my-md-0`)
+
+	rolloutValidResources = dedent.Dedent(`
+		Valid resource types include:
+
+		   * machinedeployments
```

Comment:
Since the key we are using is `machinedeployment/foo`, might be nice to keep it consistent.
```suggestion
		   * machinedeployment
```